src/backend/distributed/executor/subplan_execution.c

```diff
@@ -19,6 +19,11 @@
 #include "executor/executor.h"
 
 
+int MaxIntermediateResult = 1048576; /* maximum size in KB the intermediate result can grow to */
+uint64 currentIntermediateResult = 0;
```

Comment:
since this contains a size and it's the total size across all subplans, maybe `TotalIntermediateResultSize`